nautobot/utilities/filters.py

```diff
@@ -580,6 +587,14 @@ def add_filter(cls, new_filter_name, new_filter_field):
             cls._generate_lookup_expression_filters(filter_name=new_filter_name, filter_field=new_filter_field)
         )
 
+    @classmethod
+    def get_fields(cls):
+        fields = super().get_fields()
+        if "id" not in fields and (cls._meta.exclude is None or "id" not in cls._meta.exclude):
+            # Add "id" as the first key in the `fields` OrderedDict
+            fields = OrderedDict(id=[django_filters.conf.settings.DEFAULT_LOOKUP_EXPR], **fields)
```

Comment:
> Why does `id` need to be first?

In most of our existing filtersets it is, so I wanted to be consistent. There's no technical reason it has to be.

> What if it's already in `fields` but not first?

Then we leave it where it is, since presumably there's a reason for that.

> Also #1746: Why an OrderedDict?

Because the base FilterSet.get_fields() in django-filters returns an OrderedDict and I didn't want to change the API return type just in case there's something in django-filters that relies on that.